spec/unit/rust-crypto/rust-crypto.spec.ts

```diff
@@ -47,9 +48,52 @@ describe("RustCrypto", () => {
             rustCrypto = await makeTestRustCrypto();
         });
 
-        it("should return a list", async () => {
+        it("should import and export keys", async () => {
```

Comment:
.exportRoomKeys certainly should *not* import room keys :-p

can you rename the tests?

Reply:
this test still has a misleading name

Reply:
Updated 
> describe(".importThenExportRoomKeys", ..)